openvino_tensorflow/kernels/encapsulate_op.cc

```diff
@@ -221,11 +221,9 @@ void NGraphEncapsulateOp::Compute(OpKernelContext* ctx) {
       s_tf_timing_run_enabled_map[m_cluster_id]) {
     // Measure the timing of cluster through force TF run
     int64_t start_ns, duration_in_ms;
-    if (BackendManager::OVTFProfilingEnabled())
```

Comment:
Good catch. This would break the cost measuring for fallback. Missed that in previous PR review.